src/rdma.c

```diff
@@ -991,6 +1031,8 @@ static valkeyContextFuncs valkeyContextRdmaFuncs = {
     .async_read = valkeyRdmaAsyncRead,
     .async_write = valkeyRdmaAsyncWrite,
     .read = valkeyRdmaRead,
```

Comment:
Should we replace the handling within `valkeyRdmaRead` with an assert to avoid having unused code?

Reply:
Good idea, no need to keep the dead code.